core/domain/exp_domain.py

```diff
@@ -1924,6 +1924,26 @@ def _convert_states_v46_dict_to_v47_dict(cls, states_dict):
                 html_validation_service.convert_svg_diagram_tags_to_image_tags)
         return states_dict
 
+    @classmethod
+    def _convert_states_v47_dict_to_v48_dict(cls, states_dict):
```

Comment:
Do we also need to write the function to migrate/discard draft changelist?

Reply:
Yup, added now!